mobile/src/main/java/rs/readahead/washington/mobile/views/adapters/GalleryRecycleViewAdapter.java

```diff
@@ -157,8 +155,12 @@ private void removeAllSelections() {
 
     private void checkItemState(ViewHolder holder, VaultFile mediaFile) {
         boolean checked = selected.contains(mediaFile);
+        // Get the string resource ID based on the checked state
+        String contentDescriptionResString = holder.binding.checkBox.getContext().getString(checked ? R.string.action_unselect : R.string.action_select);
```

Comment:
As I mentioned before let's try to provide the context another way, maybe throw the adapter constructor?